src/main/java/com/google/api/generator/gapic/composer/SampleCodeHelperComposer.java

```diff
@@ -45,26 +54,49 @@ public static TryCatchStatement composeRpcMethodSampleCode(
       return composeLroUnaryRpcMethodSampleCode(method, arguments, clientType);
     }
     // Pure Unary RPC method.
-    return composeUnaryRpcMethodSampleCode(method, arguments, clientType);
+    return composeUnaryRpcMethodSampleCode(method, arguments, clientType, resourceNames);
   }
 
   private static TryCatchStatement composeUnaryRpcMethodSampleCode(
-      Method method, List<MethodArgument> arguments, TypeNode clientType) {
-    // TODO(summerji): compose sample code for unary rpc method.
+      Method method,
+      List<MethodArgument> arguments,
+      TypeNode clientType,
+      Map<String, ResourceName> resourceNames) {
+    // TODO(summerji): Add unit tests.
     VariableExpr clientVarExpr = createVariableExpr(getClientName(clientType), clientType);
+    // Assign each method arguments with default value.
+    List<Expr> bodyExpr =
+        arguments.stream()
+            .map(methodArg -> assignMethodArgumentWithDefaultValue(methodArg, resourceNames))
+            .collect(Collectors.toList());
+    // Invoke current method based on return type.
+    // e.g. if return void, echoClient.echo(..); or,
+    // e.g. if return other type, EchoResponse response = echoClient.echo(...);
+    if (method.outputType().equals(TypeNode.VOID)) {
```

Comment:
PTAL at #534. The proto equivalent of `void` is `Empty`, and #534 has examples of how to detect such types.

Reply:
Also, `deleteExclusion` is a good output example to check for this case.

Reply:
Thanks, very useful. Updated code.